addons/website_slides/controllers/main.py

```diff
@@ -348,13 +349,20 @@ def _get_top3_users(self):
         '/slides/<model("slide.channel"):channel>/tag/<model("slide.tag"):tag>',
         '/slides/<model("slide.channel"):channel>/tag/<model("slide.tag"):tag>/page/<int:page>',
         '/slides/<model("slide.channel"):channel>/category/<model("slide.slide"):category>',
-        '/slides/<model("slide.channel"):channel>/category/<model("slide.slide"):category>/page/<int:page>'
+        '/slides/<model("slide.channel"):channel>/category/<model("slide.slide"):category>/page/<int:page>',
+        '/slides/<model("slide.channel"):channel>/uncategorized',
+        '/slides/<model("slide.channel"):channel>/uncategorized/page/<int:page>',
     ], type='http', auth="public", website=True, sitemap=sitemap_slide)
-    def channel(self, channel, category=None, tag=None, page=1, slide_type=None, uncategorized=False, sorting=None, search=None, **kw):
+    def channel(self, channel, category=None, tag=None, page=1, slide_type=None, sorting=None, search=None, **kw):
+        """
+        Will return all necessary data to display the requested slide_channel along with a possible category.
+        """
         if not channel.can_access_from_current_website():
             raise werkzeug.exceptions.NotFound()
 
         domain = self._get_channel_slides_base_domain(channel)
+        regex = r"\/uncategorized[\/?]"
```

Comment:
Meh. Somehow if my channel is labelled "How to retrieve uncategorized dogs", won't it fail ?

My /uncategorized/ idea was not so good maybe. Why not a uncategorized=False/True parameter on classic channel route ? It seems like the old implementation actually, did it prove wrong ?